platform/ios/jazzy.yml

```diff
@@ -80,15 +91,12 @@ custom_categories:
       - MGLTilePyramidOfflineRegion
   - name: Geometry
     children:
```

Comment:
Let’s also add `MGLCoordinateBoundsIntersectsCoordinateBounds` to Geometry. (It’s in Other Functions, currently.)